### DISCUSSION TITLE
How to launch the GraphScope after running the /scirps/prepare_env.sh?

### DISCUSSION THREAD

**Discussion by Zarca** [alibaba/GraphScope#124] [General]
Hi,
I'm puzzled at how can I launch the GraphScope's k8s ,just after using the prepare_env.sh shell  to pull all the needed docker images like graphscope, zookeeper, etcd...then how can i launch and arrange the pods，is it should be launched and be set manually separately?? which means , I should start the GraphScope as a pod firstly , and then start the etcd or zookeeper as another pod ...?
Or, are these  works  all done by the graphscope's Session?

**Comment by siyuan0322** [comment]
> Or, are these works all done by the graphscope's Session?

Yes, `import graphscope; sess = graphscope.session()` will launch pods for you, additional parameters may be passed to manage configurations like worker number, cpu, mem, etc.

**Comment by Zarca** [comment]
Hi,
siyuan,
Thanks for reply and  Got it :)

**Comment by sighingnow** [comment]
Close as solved.

Thanks for your interest on GraphScope @Zarca. Feel free to reopen if you have any further questions.

**Comment by Zarca** [comment]
Hi @sighingnow  ,Closing the issue is ok :）
